README.md

```diff
@@ -11,53 +11,57 @@ docker compose up
 ```
 
 ## Full Project Setup
-If a databse dump (usagov.sql) is acquired from a specific hosted environment, a full copy of that environment can be created as a starting point.
+### Import SQL Database
+Safe development database dumps are kept in Google Drive. You can download and import a SQL database from https://drive.google.com/drive/folders/1zVDr7dxzIa3tPsdxCb0FOXNvIFz96dNx?usp=sharing. We recommend using the latest database available.
 
+Unzip the file and insert directly into the **root** directory.
+
+### Initialization
+**Note: Please wait until each command finishes before running the next. Expect long wait times. We recommend keep your laptop (if you're using one) plugged in during this setup.**
+
+1. Open up your IDE/terminal and run the following commands.
 ```
 bin/init
 docker compose up
+```
+
+2. In a separate terminal, run the following:
+
+```
 bin/db-update
 bin/drupal-update
+```
+
+3. Return to the original terminal where you ran `docker compose up` and type CTRL-C. 
+
+4. Run the following command:
```

Comment:
Steps 3-5  should  be unnecessary.  db-update runs inside the database container started in step 1, and drupal-update actually asks composer to start up totally new containers to handle each step. So stopping and starting composer shouldn't do anything. I think maybe it was left at the bottom as a mistake